src/tests/components/SearchForm.test.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import SearchForm from "../../components/SearchForm";
+
+describe("SearchForm", () => {
+  test("renders the input field and button", () => {
+    const { getByPlaceholderText, getByRole } = render(<SearchForm />);
+    const inputElement = getByPlaceholderText("Search Location...");
+    const buttonElement = getByRole("button");
+    expect(inputElement).toBeInTheDocument();
```

Comment:
You should include a blank line between the the end of the setup and the start of the assertions in your tests, it makes them easier to read at a glance:
```
    const buttonElement = getByRole("button");

    expect(inputElement).toBeInTheDocument();
```